### PR TITLE
Added basic tech preview support

### PR DESCRIPTION
**Pull request by dbecvarik**

### CODE REVIEW

File: concreate/cli.py [hunk 1]
```diff
@@ -61,7 +61,12 @@ def run(self):
                                  dest='build_osbs_release',
                                  action='store_true',
                                  default=False,
-                                 help='execute OSBS release build.')
+                                 help='execute OSBS release build')
+
+        parser.add_argument('--tech-preview',
```
Comment (goldmann) [inline concreate/cli.py]: Shouldn't this be `--build-tech-preview`? It's a build flag.

File: concreate/cli.py [hunk 2]
```diff
@@ -61,7 +61,12 @@ def run(self):
...
+                            action='store_true',
+                            default=False,
```
Comment (goldmann) [inline concreate/cli.py]: False is default by default :)

File: concreate/cli.py (hunk 1 above)
Comment (dbecvarik) [reply concreate/cli.py]: i'm not sure its basically generate option - but this should be hidden in future - so Im not sure here

File: concreate/cli.py (hunk 2 above)
Comment (dbecvarik) [reply concreate/cli.py]: good point

File: concreate/cli.py (hunk 1 above)
Comment (goldmann) [reply concreate/cli.py]: You even added it to the build documentation...

File: docs/build.rst [hunk 3]
```diff
@@ -23,7 +23,7 @@ You can execute image build by running:
 * ``--build-engine`` -- a builder engine to use ``osbs`` or ``docker``
 * ``--build-tag`` -- an image tag used for build image (can be specified multiple times)
 * ``--build-osbs-release`` -- perform a OSBS release build
-
+* ``--build-tech-preview`` -- updates JBOSS_IMAGE_NAME variable to contain '-tech-preview' suffix in family part of the image name
```
Comment (goldmann) [inline docs/build.rst]: Do not mention here that variable. Important is only that the name, before slash will be suffixed.

**Pull request merged**